RocketCast/XMLParser.swift

```diff
@@ -72,6 +75,9 @@ extension XMLParser: XMLParserDelegate {
                         tmpEpisode!.title = information
                     }
                 } else  {
+                    if (DatabaseController.doesThisPodcastAlreadyExist(podcastTitle: information)) {
```

Comment:
We wont need this code anymore, by setting the mergePolicy in databaseController.